guides/Other Subjects/PieceStores.md

```diff
@@ -33,122 +33,50 @@ class Something extends Piece {
 
 ```javascript
-const { Piece, util } = require('klasa');
-const getInfoAsync = require('util').promisify(require('ytdl-core').getInfo);
-
-class Genre extends Piece {
-
-	/**
-	 * @param {KlasaClient} client The Klasa Client
-	 * @param {GenreStore} store The Genre store
-	 * @param {Array} file The path from the pieces folder to the command file
-	 * @param {boolean} core If the piece is in the core directory or not
-	 * @param {Object} [options={}] Optional Genre settings
-	 */
-	constructor(client, store, file, core, options = {}) {
-		super(client, store, file, core, options);
-
-		// we should probably describe our auto play genres in the command,
-		// we will make for guild owners to set their guild's genre setting.
-
-		/**
-		 * The genre's description
-		 * @type {string}
-		 */
-		this.description = options.description || '';
-
-		// Some example artists to include with the descriptions
-
-		/**
-		 * Examples of artists for this genre
-		 * @type {string[]}
-		 */
-		this.examples = options.examples || [];
-	}
+const { Piece } = require('klasa');
 
-	// We don't even need run for this, let's call it getNext
-	async getNext(player) {
-		// Let's define a default behavior here
-		// If we aren't playing anything yet, get a link from this.seeds
-		// (which is defined in each extension piece)
-		if (!player.playingURL) return this.wrapLink(this.seeds[Math.floor(Math.random() * this.seeds.length)]);
-
-		// If we do have a link, lets get youtube info about that link
-		const info = await getInfoAsync(player.playingURL).catch((err) => {
-			this.client.emit('log', err, 'error');
-			throw `something happened with YouTube URL: ${player.playingURL}\n${util.codeBlock('', err)}`;
-		});
-
-		// Find the first video that we haven't recenly played on our player
-		const next = info.related_videos.find(vid => vid.id && !player.recentlyPlayed.includes(this.wrapLink(vid.id)));
-		// If their isn't a video, reseed a video we havn't played recently
-		if (!next) {
-			const seed = this.seeds.find(vid => !player.recentlyPlayed.includes(this.wrapLink(vid)));
-			// If we have played all of the seeds, start over on reseeding
-			if (!seed) {
-				player.recentlyPlayed = [];
-				return this.wrapLink(this.seeds[Math.floor(Math.random() * this.seeds.length)]);
-			}
-
-			// Else return the seed we haven't played recently
-			return this.wrapLink(seed);
-		}
+class RawEvent extends Piece {
 
-		// Otherwise wrap the next song
-		return this.wrapLink(next.id);
-	}
+	constructor(client, store, file, core, options = {}) {
+		super(client, store, file, core, options);
 
-	wrapLink(id) {
-		return `https://youtu.be/${id}`;
+		// not a lot of options, but im keeping it simple for the sake of the guide
```

Comment:
I mean this comment could be improved to be like `// Enable this raw event by default if not specifically disabled in the options`